conv/abci/block_test.go

```diff
@@ -0,0 +1,191 @@
+package abci
+
+import (
+	cmbytes "github.com/cometbft/cometbft/libs/bytes"
+
+	cmproto "github.com/cometbft/cometbft/proto/tendermint/types"
+
+	"testing"
+	"time"
+
+	cmversion "github.com/cometbft/cometbft/proto/tendermint/version"
+	cmtypes "github.com/cometbft/cometbft/types"
+
+	"github.com/rollkit/rollkit/types"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var header = &types.Header{
+	BaseHeader: types.BaseHeader{
+		Height:  12,
+		Time:    uint64(time.Now().Local().Day()),
+		ChainID: "test",
+	},
+	Version: types.Version{
+		Block: 1,
+		App:   2,
+	},
+	LastHeaderHash:  types.GetRandomBytes(32),
+	LastCommitHash:  types.GetRandomBytes(32),
+	DataHash:        types.GetRandomBytes(32),
+	ConsensusHash:   types.GetRandomBytes(32),
+	AppHash:         types.GetRandomBytes(32),
+	LastResultsHash: types.GetRandomBytes(32),
+	ProposerAddress: types.GetRandomBytes(32),
+	AggregatorsHash: types.GetRandomBytes(32),
+}
```

Comment:
Instead of introducing global variables, can we pull into helper functions like this?
```suggestion
func getRandomHeader() *types.Header {
        return &types.Header{
	        BaseHeader: types.BaseHeader{
		        Height:  12,
		        Time:    uint64(time.Now().Local().Day()),
		        ChainID: "test",
	        },
	        Version: types.Version{
		        Block: 1,
		        App:   2,
	        },
	        LastHeaderHash:  types.GetRandomBytes(32),
	        LastCommitHash:  types.GetRandomBytes(32),
	        DataHash:        types.GetRandomBytes(32),
	        ConsensusHash:   types.GetRandomBytes(32),
	        AppHash:         types.GetRandomBytes(32),
	        LastResultsHash: types.GetRandomBytes(32),
	        ProposerAddress: types.GetRandomBytes(32),
	        AggregatorsHash: types.GetRandomBytes(32),
        }
}
```